recipes/tzlocal/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "1.1.2" %}
+
+package:
+  name: tzlocal
+  version: {{ version }}
+
+source:
+  fn: tzlocal-{{ version }}.tar.gz
+  url: https://github.com/regebro/tzlocal/archive/{{ version }}.tar.gz
+  sha256: 939de4cf9535f61a5c1a99f34a447e3992f8046fd1bedeac9ec7892b419cf963
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytz
+
+  run:
+    - python
+    - pytz
+
+test:
+  imports:
+    - tzlocal
+  requires:
+    - setuptools
+  commands:
+    - cd {{ environ.SRC_DIR }} && python setup.py test
```

Comment:
I didn't think that `test` had a `script` section.  At least it doesn't appear to in the conda-build docs.  Should i just remove this part that executes the unit tests?